src/app/utils/analytics-datalayer.js

```diff
@@ -0,0 +1,37 @@
+// @ts-check
+
```

Comment:
Can we leave a comment here pointing to the upstream version of this code in case we ever need to refer back?

Reply:
Good idea